### DISCUSSION TITLE
post/nju-cyber-run/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#6] [Announcements]
# post/nju-cyber-run/

仅供安卓逆向交流学习, 相关数据已脱敏.             最近某高校推出了要求学生跑步打卡的操作, 虽然配速要求其实并不严格, 但一学期要打卡 24 次, 每次至少 2400 m… 对我来说还是有点困难了, 所以来研究一下相关的软件. 0x00 前期观察&信息收集通过分析相关 Activity 的包名发现这个跑步打卡的部分并不是学校自己做的, 是整合了一个

https://blog.lyc8503.site/post/nju-cyber-run/

**Comment by Maroon1989** [comment]
我超太强了

**Comment by xing-junyang** [comment]
tql~~

**Comment by ShelfSeeck** [comment]
卧槽tql
